fuse_core/src/async_publisher.cpp

```diff
@@ -48,43 +48,56 @@ AsyncPublisher::~AsyncPublisher()
   internal_stop();
 }
 
-void AsyncPublisher::initialize(const std::string & name)
+void AsyncPublisher::initialize(
+  node_interfaces::NodeInterfaces<
+    node_interfaces::Base,
```

Comment:
Nit, I think this could use `ALL_RCLCPP_NODE_INTERFACES` macro. The macro says `fuse_core::...`, but I think that will still work inside the `fuse_core` namespace.

Reply:
Ah I missed this one